benchmark.txt

```diff
@@ -0,0 +1,253 @@
+PUBLISH_BATCH_SIZE      Requests        Workers Time    Average Time
```

Comment:
benchmark summary for PUBLISH_BATCH_SIZE = 100:
|PUBLISH_BATCH_SIZE|	Requests|	Workers|	Time/s|	Average Time/s|
|---|---|---|---|---|
|100	|1000|	16|	2.92	|0.0029|
|100|	1000|	4|	7.28|	0.0073|
|100|	1000|	2|	9.26|	0.0093|
|100|	1000|	10|	19.18|	0.0192|
|100|	1000|	8|	21.06|	0.0211|
|100|	1000|	1|	29.4|	0.0294|
|100|	1000|	12|	30.77|	0.0308|